cli/assets.go

```diff
@@ -100,7 +101,8 @@ func listAllAssetsCommand() *cobra.Command {
 
 func makeGetAllAssetRequest(types, services, data, q, sort, sort_dir string, size, page uint32) *compassv1beta1.GetAllAssetsRequest {
 	newReq := &compassv1beta1.GetAllAssetsRequest{
-		Size: size,
+		Size:        size,
+		NamespaceId: namespace.DefaultNamespace.ID.String(),
```

Comment:
It may be unnecessary, it's not that we are allowing users to list assets from multiple namespaces, if all of them belong to the same namespace should we really add another column for it? Wdyt?

Reply:
make sense